osf/models/licenses.py

```diff
@@ -8,11 +8,11 @@
 
 def _serialize(fields, instance):
     return {
-        field: getattr(instance, field)
+        field: getattr(instance, field if field != 'id' else 'license_id')
         for field in fields
     }
 
-serialize_node_license = functools.partial(_serialize, ('_id', 'name', 'text'))
+serialize_node_license = functools.partial(_serialize, ('id', 'name', 'text'))
```

Comment:
Frontend code expects `license_id` to be serialized as `id`. I can't say exactly why, but it does.